### ISSUE TITLE
Miss loss_GRPO

### ISSUE THREAD

**Issue by leotompson** [AIGeeksGroup/3D-R1#33]
Your work is very impressive, and I would like to follow your work in industrial scenarios. Could you please tell me where the calculation of loss_GRPO is? I can only find policy_loss in the code, which seems to be inconsistent with the calculation of GRPO.
```
        # Compute KL divergence
        kl_div = self.compute_kl_divergence(current_logits, ref_logits)
        
        # Compute policy loss with KL penalty
        policy_loss = -torch.mean(rewards) + self.config['beta'] * kl_div
        
        # Backward pass
        self.optimizer.zero_grad()
        policy_loss.backward()
        
        # Gradient clipping
        torch.nn.utils.clip_grad_norm_(
            self.model.parameters(), 
            self.config['max_grad_norm']
        )
        
        self.optimizer.step()
```

**Comment by leotompson** [comment]
Thank you for your response. Based on your code, I applied it to my downstream task. I understand that in the formula `policy_loss = -torch.mean(rewards) + self.config['beta'] * kl_div`, rewards is a vector with gradients, and kl_div should also have gradients for the penalty. 
However, both `logits_current` and `logits_ref` are computed `with torch.no_grad()`, which led to the issue in my project where kl_div does not have gradients. As a result, kl_div will not lead to model updates.

Once again, thank you for your reply, it has deepened my understanding and thought process in this area.
